packages/react-ui-components/src/stories/Components/StateIndicator.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { KvStateIndicator } from '../../components';
+
+KvStateIndicator.displayName = 'KvStateIndicator';
```

Comment:
@ricardogoncalves89 should the name be dash cased or pascal cased?

@ricardoarasilva

Reply:
pascal cased